data/repairable_items.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
These should be the item ids, avoid display names

Reply:
wouldnt names be better?

Reply:
The display names are more likely to change/break than the item ids, and they're easier to map later on if needed

Reply:
> The display names are more likely to change/break than the item ids, and they're easier to map later on if needed

gotcha alright